src/concurrently.ts

```diff
@@ -179,6 +185,10 @@ export function concurrently(
         new ExpandWildcard(),
     ];
 
+    if (options.matrices?.length) {
```

Comment:
Looks like matrices have preference over passthrough arguments using the same `{number}` syntax. Was it intended?

It needs to be called out in the help and in the docs.